docs/setup/deploying-a-runtime.md

```diff
@@ -0,0 +1,294 @@
+# Deploying a Runtime
+
+Before proceeding, make sure to look at the [prerequisites] required for running
+an Oasis Core environment followed by [build instructions] for the respective
+environment (non-SGX or SGX), using the [oasis net runner] and see
+[runtime documentation] for a general documentation of runtimes.
+
+These instructions will show how to register and deploy a runtime node on a
+local development network.
+
+[prerequisites]: prerequisites.md
+[build instructions]: building.md
+[oasis net runner]: oasis-net-runner.md
+[runtime documentation]: ../runtime/index.md
+
+## Provision a single validator node network
+
+Use the [oasis net runner] to provision a validator node network without any
+registered runtime.
+
+```
+mkdir /tmp/runtime-example
+
+# TODO: this json fixture will probably be hard to keep up to date. Figure
```

Comment:
I fixed this by extending the `default fixture` command, and using that here instead of providing a separate json, which should be much better in terms of rotting, but still not perfect.

I think docs will generally tend to rot (specially the more low level ones) unless someone periodically goes over the docs and actually tries them out. Maybe adding a "make sure documentation is up to date" should be added to the process of creating major releases (as a manual step). However we do have quite a lot of docs already in this repository, so the exact step should probably be something like: "documentation owners" should make sure documentation is up to date with the version getting released - few days before releasing a new major version?  ("documentation owners" - since it's probably a bit much for a single person to go over all).